senders/client.go

```diff
@@ -164,21 +164,15 @@ func (sender *wavefrontSender) SendSpan(name string, startMillis, durationMillis
 func makeSpanTags(tags []SpanTag) []span.Tag {
 	var spanTags []span.Tag
```

Comment:
Since you know what the length of your returned slice will be ahead of time, you could just say ```spanTags := make([]span.Tag, len(tags))``` here.  Doing that will save Go from having to resize your slice as you add elements to it.